hazelcast/src/main/java/com/hazelcast/internal/metrics/MetricsRegistry.java

```diff
@@ -196,4 +196,9 @@
      * @param objects the array of objects to check.
      */
     void discardMetrics(Object... objects);
+
+    /**
+     * Creates a new {@link ProbeBuilder}.
+     */
+    ProbeBuilder newProbeBuilder();
```

Comment:
Goal is to make probing more straight forward and uniform not more diverse. So this new API is most likely subject to change.

Reply:
Can you propose a specific way of making it "more straightforward"?

Reply:
I also don't see a problem with ProbeBuilder. I even suggested it to @viliam-durina to prevent ending up with methods using many arguments on the MetricsRegistry. So what should be changed?